Update readme.md and other fixes (#359)
yiisoft/di: README.md

$container = new Container($config);
```

> # Benchmarks

Reviewer: That is for internals.

Reviewer 2: @luizcmarin Benchmarks for internals

(best [mean mode] worst) = 0.113 [4,483.856 4,486.051] 0.117 (μs) 
⅀T: 246,612.096μs μSD/r 43.563μs μRSD/r: 1.336%
```

> **Warning!**
> These summary statistics can be misleading.
> You should always verify the individual subject statistics before drawing any conclusions.

> >

Reviewer 2: ```suggestion
```

Not need.

>   * stdev: Standard deviation.
>   * rstdev: The relative standard deviation.
>   * diff: Difference between variants in a single group.
```

## Documentation

- [Internals](docs/internals.md)

> ## Support
> 

Reviewer 2: ```suggestion
```

[![Telegram](https://img.shields.io/badge/telegram-join-1DA1F2?style=flat&logo=telegram)](https://t.me/yii3en)
[![Facebook](https://img.shields.io/badge/facebook-join-1DA1F2?style=flat&logo=facebook&logoColor=ffffff)](https://www.facebook.com/groups/yiitalk)
[![Slack](https://img.shields.io/badge/slack-join-1DA1F2?style=flat&logo=slack)](https://yiiframework.com/go/slack)

## License

> The Yii DI is free software. It is released under the terms of the BSD License.

Reviewer 2: ```suggestion
The Yii Dependency Injection is free software. It is released under the terms of the BSD License.
```
